### ISSUE TITLE
ConfirmDialog shows double icon in confirm button

### ISSUE THREAD

**Issue by StephanBis** [primefaces/primeng#13108]
### Describe the bug

After updating Angular and PrimeNG to to v15, the ConfirmDialog shows two icons when you set a custom icon. When not defining a custom icon, the default 'checkmark' icon is shown only once, as expected.

![image](https://github.com/primefaces/primeng/assets/10954823/a81340bf-34ee-4b93-b1d2-99ddaf3729a9)

I'm using the ConfirmationService like this:
```
this.confirmationService.confirm({
    message: message,
    header: header,
    icon: 'pi pi-exclamation-triangle',
    acceptLabel: this.translateService.instant('general.yes'),
    acceptIcon: 'pi pi-trash',
    acceptButtonStyleClass: 'p-button-danger',
    rejectLabel: this.translateService.instant('general.no'),
    rejectButtonStyleClass: 'p-button-secondary p-button-text',
    accept: accept,
    defaultFocus: 'none'
});
```

### Environment

- angular: ^15.2.9
- primeflex: ^3.3.0
- primeicons: ^6.0.0
- primeng: ^15.0.0

### Angular version

15.2.9

### PrimeNG version

15.0.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.18.1

**Comment by volvachev** [comment]
Hi, it's duplicate #13001

**Comment by nelioyann** [comment]
Could not find a decent workaround so I decided to hide those icons wherever unnecessary
`acceptIcon: "hidden", rejectIcon: "hidden",`
You could also pass a class for hiding one of those icons but this will most likely require a refactoring once this issue is fixed

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
